account/utils.py

```diff
@@ -0,0 +1,31 @@
+import json
+import requests
+import jwt
+
+from django.http import JsonResponse
+from django.core.exceptions import ObjectDoesNotExist
+
+from chanel.settings import SECRET_KEY, ALGORITHM
+from .models import Account
+
+def login_decorator(func):
+    def wrapper(self, request, *args, **kwargs):
+        token = request.headers.get('Authorization', None)
+        
+        if token is not None:
```

Comment:
```suggestion
        if token:
```
not None과 동일한 의미입니다.